device/accton/x86_64-accton_as7212_54x-r0/AS7212-54x/config_db.json

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
config_db.json is generated by minigraph.xml, we do not need this in the source tree.